src/ActionModal/components/TableInvoice.vue

```diff
@@ -1,20 +1,25 @@
 <template>
   <div>
     <ul class="table-invoice">
-      <li v-if="subTotal > 0">
+      <li v-if="subTotal > 0" class="sub-total">
         <span>Subtotal</span>
         <span> {{ subTotal | fullDecimals }} {{ bondDenom }} </span>
       </li>
-      <li>
+      <li class="fees">
         <span>Network Fee</span>
         <span>
           {{ estimatedFee | fullDecimals }}
-          {{ bondDenom }}
+          {{ feeDenom || bondDenom }}
         </span>
       </li>
       <li class="total-row">
         <span>Total</span>
-        <span> {{ total | fullDecimals }} {{ bondDenom }} </span>
+        <div class="total-column">
+          <p>{{ total | fullDecimals }} {{ bondDenom }}</p>
+          <p v-if="feeDenom">
```

Comment:
```suggestion
          <p v-if="feeDenom !== bondDenom">
```

Reply:
Shouldn't it rather be `feeDenom && feeDenom !== bondDenom`?

Reply:
feeDenom defaults to `""`

Reply:
yes your right